.github/CONTRIBUTING.md

```diff
@@ -7,7 +7,6 @@ Any typo fixes, spelling fixes, fixes in documentation,
 or README.md can be done without any further considerations!
 Simply do the fixes in your cloned repo and create a pull request.
 
-
```

Comment:
I kind of like two new-lines in front of titles. Is it possible to customize this?

Reply:
> I kind of like two new-lines in front of titles.

Yeah, I noticed that 😊. I am not a big fan of it though, especially since my editor already highlights sections:

<img width="516" alt="contributing" src="https://user-images.githubusercontent.com/691989/52909201-d610ae00-3284-11e9-86d7-2383738cd6c7.png">

.

> Is it possible to customize this?

Not as far as I can tell. At least the [option page for prettier](https://prettier.io/docs/en/options.html) does not list such an option.

Prettier does use [remark-parse](https://github.com/remarkjs/remark/tree/master/packages/remark-parse) though, which might provide such an option. remark(-parse) seems pretty powerful, since it even provides a [plugin-system](https://github.com/remarkjs/remark/blob/master/doc/plugins.md). However, the documentation of the tool is pretty bad in my opinion. I was not able to find an option to add multiple newlines before a heading.

Reply:
Thank you for looking it up! I will be okay with a single new-line.